stories/topics/proc-aws-generate-remove-data-files.adoc

```diff
@@ -14,15 +14,20 @@ $ mkdir command_generator_data
 ----
 . Populate the `$(pwd)/command_generator_data` folder with the configuration file template.
 +
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:con-tech-note-linux-files-owned-by-root[technical notes] for more information.
```

Comment:
generator is owned by...

For more information, read xref:con-tech-note-linux-files-owned-by-root[technical notes].